_redirects

```diff
@@ -1,16 +1,37 @@
+# add new redirects to the top because they will override later ones
+# getting started guide
+/guides/getting-started /docs/get-started/getting-started/overview 301
+/docs/get-started/getting-started/create-a-project  /docs/get-started/getting-started/building-your-first-project/build-your-first-models 301
```

Comment:
sugg: 

the project creation content is at https://docs.getdbt.com/docs/get-started/getting-started/getting-set-up/setting-up-bigquery#initialize-your-dbt-project. it's not good practice to redirect to an anchor link so maybe consider "About set up"  https://docs.getdbt.com/docs/get-started/getting-started/set-up-dbt-cloud